.gitattributes

```diff
@@ -1,6 +1,5 @@
 # `git apply` and friends don't understand CRLF, even on windows. Force those
 # files to be checked out with LF endings even if core.autocrlf is true.
-*.patch text eol=lf
```

Comment:
Can we fix this without deleting this?

Reply:
Closing in re https://github.com/electron/electron/pull/45704#discussion_r1967976957